documentation/commands/test.md

```diff
@@ -6,6 +6,7 @@ Usage: `exo test`
 
 - runs the individual tests for all services
 - runs the end-to-end tests for the application
+- `exo generate docker-compose` is run at the beginning of this command
 
-Tests for services are defined in the [service configuration](),
-tests for the entire application in the [application configuration]().
+Tests for services are defined in the [service configuration](documentation/configuration_files/service.md),
+tests for the entire application in the [application configuration](documentation/configuration_files/application.md).
```

Comment:
I think line 12 should be removed as we don't have tests for the entire application yet